### PR TITLE
EZS-768: Implement choice location while creating content

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZS-768

**Screencast:**
cof-select-location

**TO DO:**
- [x] implemented edit location

### CODE REVIEW

**Comment by dew326** [comment]
ping @lserwatka @sunpietro 
@StephaneDiot - Could you review my work?

File: Resources/public/js/views/cof-contentcreationview.js [hunk 1]
```diff
@@ -171,6 +194,57 @@ YUI.add('cof-contentcreationview', function (Y) {
...
+            var eventNewVal = event.newVal,
+                locationPath,
+                contentInfo,
+                selectedName,
+                pathSeparator = '/',
+                selectedPath = pathSeparator;
+
+            contentInfo = eventNewVal.contentInfo;
+            locationPath = eventNewVal.location.get('path');
+            selectedName = contentInfo.get('name');
```
Comment (StephaneDiot) [inline Resources/public/js/views/cof-contentcreationview.js]: niptick: you should initialize those 3 variables in their declaration

**Comment by dew326** [comment]
ping @StephaneDiot

**Comment by StephaneDiot** [comment]
well I have trouble adding your bundle... added `new EzSystems\EzContentOnTheFlyBundle\EzSystemsEzContentOnTheFlyBundle(),` to AppKernel.php and cloned your repo but I'm getting an error 

> Fatal error: Class 'EzSystems\EzContentOnTheFlyBundle\EzSystemsEzContentOnTheFlyBundle' not found

Am I missing something?

**Comment by StephaneDiot** [comment]
but from what I see it seems ok to me

**Comment by dew326** [comment]
I asked @kamilmusial and he suggests that after cloning repo into `src\EzSystems\EzContentOnTheFlyBundle` and adding to AppKernel.php, you should also run `composer dump-autoload` in the terminal.

**Comment by andrerom** [comment]
> autoload issue

We should add this on packagist today or tomorrow and make a first tag on thursday to avoid that issue.

**Pull request merged**